### PR TITLE
feat (Pages design): Workspace, Program, Auth pages

### PR DESCRIPTION
**Pull request by pipesorter**

### CODE REVIEW

File: packages/ui/components/AppLayout/AppLayout.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ const AppLayout = ({ children }: ChildrenProps) => {
   return (
     <div className="flex">
       <SideBar />
-      <div className="w-full min-h-screen">
+      <div data-testid="121" className="w-full min-h-screen">
```
Comment (pipesorter) [inline packages/ui/components/AppLayout/AppLayout.tsx:16]: Would need to remove this @pipesort123TP

File: packages/ui/components/AppLayout/Navbar.tsx [hunk 2]
```diff
@@ -5,80 +5,125 @@ import { Button } from 'components/Button';
...
+  const [hover, setHover] = useState(false);
+
+  const tooltipStyle = {
+    display: hover ? 'block' : 'none',
+  };
 
   return (
-    <nav className="bg-white py-4 px-8 h-16 flex items-center justify-between">
-      <ul className="flex items-center">
+    <nav className="bg-white py-4 px-8 h-16 flex items-center justify-between border border-gray-100 hover:border-blue-400">
```
Comment (pipesorter) [inline packages/ui/components/AppLayout/Navbar.tsx:27]: @pipesort123TP would have to extract out this as a separate class under a module.css

File: packages/ui/components/AppLayout/Navbar.tsx [hunk 3]
```diff
@@ -5,80 +5,125 @@ import { Button } from 'components/Button';
...
+              size="sm"
+              color="primary"
+              className="text-sm text-red-300 inline-block h-6 w-6 rounded-full ring-2 ring-white"
+            />
+            <LetterPic
+              letter="+3"
+              size="sm"
+              color="secondary"
+              className="text-sm mr-2 inline-block h-6 w-6 rounded-full ring-2 ring-white"
+              textColor="purple"
```
Comment (pipesorter) [inline packages/ui/components/AppLayout/Navbar.tsx:124]: @pipesort123TP can we have this as an array and loop through them?

File: packages/ui/components/Pages/Program/TableRow.tsx [hunk 4]
```diff
@@ -12,43 +13,101 @@ type TableRowProps = {
...
+  let statusClass = 'status-not-started';
+  if (selectedStatus === 'completed') {
+    statusClass = 'status-completed';
+  }
+
+  if (selectedStatus === 'in-progress') {
+    statusClass = 'status-in-progress';
+  }
+  if (selectedStatus === 'not-started') {
+    statusClass = 'status-not-started';
```
Comment (pipesorter) [inline packages/ui/components/Pages/Program/TableRow.tsx:34]: @pipesort123TP  why do we have to prefix like this one, can we remove `status-` prefix please?

File: packages/ui/components/Pages/Workspace/Multipleprogram.tsx [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+                    size="sm"
+                    color="primary"
+                    className="text-sm text-gray-600 inline-block h-6 w-6 rounded-full ring-2 ring-white"
+                  />
+                  <LetterPic
+                    letter={
+                      <img
+                        className="inline-block h-6 w-6 rounded-full ring-transparent"
+                        src="https://images.unsplash.com/photo-1500648767791-00dcc994a43e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2.25&w=256&h=256&q=80"
+                        alt=""
```
Comment (pipesorter) [inline packages/ui/components/Pages/Workspace/Multipleprogram.tsx:34]: @pipesort123TP need to extract out as an hardcoded array, and should be replaced later by backend data

File: packages/ui/pages/workspace/index.tsx [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+              : 'Let us see what you got'
+          }
+        />
+        {isFirstTimeVisitor ? (
+          <div className="px-4 flex justify-center mt-8">
+            <div className="bg-white w-full py-20 flex flex-col justify-center items-center">
+              <div>
+                <img
+                  src="/OPenGT Illustrations-25 1@2x.png"
+                  width={400}
```
Comment (pipesorter) [inline packages/ui/pages/workspace/index.tsx:28]: @pipesort123TP need to downcase the name without any spaces any referenced static files.

File: packages/ui/pages/workspace/stepform.tsx [hunk 7]
```diff
@@ -0,0 +1,489 @@
...
+                      </p>
+                    </div>
+                  </div>
+                ))}
+            </div>
+          </div>
+        </div>
+      </div>
+    </AppLayout>
+  );
```
Comment (pipesorter) [inline packages/ui/pages/workspace/stepform.tsx:486]: @pipesort123TP can we pleas extract out multiple components just like we did for `pages/index.txs` ?

File: packages/ui/components/Pages/Program/EditProgramModal.tsx [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+        <InputField
+          label="Program Name"
+          value="SSMD Gene Therapy"
+          className="text-sm text-gray-500 text-light mt-8"
+          name="Program Name"
+          placeholder=""
+          reference={inputRef}
+        />
+
+        {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
```
Comment (brettstack) [inline packages/ui/components/Pages/Program/EditProgramModal.tsx:31]: Please fix or add a code comment why this is disabled

**Review by brettstack — CHANGES_REQUESTED** [review]
This is excellent work! My only comment is on accessibility

**Comment by pipesorter** [comment]
@brettstack @sanathkr can you please review and merge this p.r? 🙏

**Comment by pipesorter** [comment]
@brettstack Oh ya, regarding accessibility, had added the eslint rule through jsx-a11y library, I see some of the rules were disabled, will fix and push now. Will come back to accessibility with `WAVE` tool and fix other issues, for now will do the basic implementation like having the label etc that the library (jsx-a11y) requires.

**Comment by pipesorter** [comment]
@brettstack done, fixed accessibility-related eslint issues. 🚀

**Review by brettstack — APPROVED** [review]

**Comment by brettstack** [comment]
Well done! 🤜

**Pull request merged**